task-5/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect, useState } from 'react';
+
+function App() {

```

Comment:
```suggestion
const App = () => {
```
Not a must, but let's use arrow functions to avoid mixing both of them in the project